__tests__/app.test.js

```diff
@@ -112,11 +112,29 @@ describe('/api/articles/:article_id/comments', () => {
             return request(app)
                 .get('/api/articles/9/comments')
                 .expect(200)
-                .then(({body}) => {
-                    const {comments} = body
+                .then(({ body }) => {
+                    const { comments } = body
                     expect(comments.length).toBe(2)
```

Comment:
the other happy path to check would be if a given valid, existing article doesnt have any comments (also have a think about why this would be a happy path :D)